react/personal_project_vite/src/pages/ListPage.jsx

```diff
@@ -62,15 +62,14 @@ export function ListPage() {
                         <li>LOSE IT</li>
                     </div>
                     <div>
-                        <li>- - - - - - - - -</li>
+                        <li>{"<><><><><><><><><><>"}</li>
```

Comment:
There may be a JS library that you can add to your React project with some built in designs